### DISCUSSION TITLE
求助

### DISCUSSION THREAD

**Discussion by MoQingkong** [the1812/Bilibili-Evolved#4925] [提问 / Q&A]
使用了该插件发现双击变成了暂停而不是全屏了，找了发现没看到在哪边设置，想问问各位大佬们

**Comment by festoney8** [comment]
看一下是不是开了 **启用双击控制**

![image](https://github.com/user-attachments/assets/eb57cba7-016a-4fa8-b3b8-38921ae89cb5)
